unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ECSRuntime/Components/ECSValueTypeComponentPool.cs

```diff
@@ -0,0 +1,12 @@
+namespace DCL.ECSRuntime
+{
+    public class ECSValueTypeComponentPool<ModelType> : IECSComponentPool<ModelType> where ModelType : struct
```

Comment:
You can't really pool value types. I believe it should not never exist

Reply:
That's right, but I needed this to inherit from IECSComponentPool, because of the multiple abstraction layers already in place.
In fact, these Get and Release are never called because Internal ECSComponent (aka structs) are never called because we check if they are an ECSComponent with or without an instantiated pool.